app/components/ui/Navigation.jsx

```diff
@@ -19,40 +24,65 @@ class Navigation extends React.Component {
     }
     return false;
   }
+
+  toggleSecondarySearch() {
+    this.setState(({ showSecondarySearch }) => ({ showSecondarySearch: !showSecondarySearch }));
+  }
+
   render() {
     return (
-      <nav className="site-nav">
-        <div className="nav-left">
-          <Link className="nav-logo" to={'/'}>
-            <img src={images.logoSmall} alt="Ask Darcel" />
-          </Link>
-          {this.props.showSearch &&
+      <nav className={styles.siteNav}>
+        <div className={styles.primaryRow}>
+          <div className={styles.navLeft}>
+            <Link className={styles.navLogo} to={'/'}>
+              <img src={images.logoSmall} alt="Ask Darcel" />
+            </Link>
+            {this.props.showSearch &&
+              <form
+                onSubmit={this.submitSearch}
+                className={`${styles.navSearch} search-container form-row`}
+                role="search"
+              >
+                <input
+                  ref={(c) => { this.searchComponent = c; }}
+                  type="text"
+                  className={styles.searchField}
+                  placeholder="Search for a service or organization"
+                  name="srch-term"
+                  id="srch-term"
+                />
+              </form>
+            }
+          </div>
+          <div className={styles.mobileNavigation}>
+            <button className={styles.searchButton} onClick={this.toggleSecondarySearch} />
+            <button className={styles.hamburgerButton} onClick={this.props.toggleHamburgerMenu} />
+          </div>
+        </div>
+        <div className={`${styles.secondaryRowWrapper} ${this.state.showSecondarySearch ? '' : styles.hide}`}>
+          <div className={styles.secondaryRow}>
             <form
               onSubmit={this.submitSearch}
-              className="nav-search search-container form-row"
               role="search"
             >
               <input
-                ref={(c) => { this.searchComponent = c; }}
+                className={styles.secondarySearchField}
+                ref={c => { this.searchComponent = c; }}
                 type="text"
-                className="search-field"
+                /* TODO: update placeholder text to include dynamic number of resources */
```

Comment:
Were you going to address this TODO in this commit or is this for another one? If it is for another, could you create a ticket for it and put the ticket number in here?